docs/howtos/cut-a-new-release.md

```diff
@@ -1,17 +1,20 @@
 # Application Services Release Process
 
 ## Make a new release from latest main.
+1. Validate your changes for consumers.
+    - Smoketest your changes in iOS, android-components, and fenix to ensure the release will integrate cleanly with key downstream consumers
+      - Run the `./automation/smoke-test-firefox-ios.py` script to test integration with Firefox for iOS.
+      - Run the `./automation/smoke-test-android-components.py` script to test integration with Android Components.
+      - Run the `./automation/smoke-test-fenix.py` script to test integration with Fenix.
 
-1. Smoketest whether the release will integrate cleanly with key downstream consumers:
-    - Run the `./automation/smoke-test-firefox-ios.py` script to test integration with Firefox for iOS.
-    - Run the `./automation/smoke-test-android-components.py` script to test integration with Android Components.
-    - Run the `./automation/smoke-test-fenix.py` script to test integration with Fenix
-    - Build and run the [sync sample app](https://github.com/mozilla-mobile/android-components/tree/master/samples/sync) in android-components
-      against your local checkout using the [local publishing flow](./locally-published-components-in-fenix.md).
-
-    If these tests fail, it indicates a breaking change. Check that such a change is called out
+      **Note:** If these tests fail, it indicates a breaking change. Check that such a change is called out
     in the changelog, and file downstream bugs to track the work of integrating it.
-2. Run the `./automation/prepare-release.py` script to create a release commit and open a pull-request. If this script does not work for you, please refer to the "Create a release commit manually" section in this document.
+    - Run the android-components [sync sample app](https://github.com/mozilla-mobile/android-components/tree/master/samples/sync) to ensure sync functionality hasn't been adversely impacted using the [local publishing flow](./locally-published-components-in-fenix.md).
```

Comment:
Are there any instructions for how to run the app and what to do with it?  Is this necessary if we have the other tests running?

Reply:
For posterity, I've moved the smoke test steps to the pull request template as the responsibility of ensuring that breaking changes are identified and addressed should belong to the person who created the changes rather than the person cutting a release (particularly if the release is urgent).

Running the sync sample app (which is currently used by android components to validate app services releases) serves as an end-to-end test. The smoke tests are generally app services related unit tests in consumer repos. So in theory running the sync sample app provides value. In practice, this step is too hand-wavy to provide consistently meaningful value.

Also I highly doubt this is a step that's always executed or that, when it is executed, it provides enough value to warrant the effort. At minimum, if we decide to keep this step, we should explicitly state how the app should be configured (particularly which emulators to use) and what should be tested once the app is running.